public/app/features/logs/components/panel/LogLineContext.tsx

```diff
@@ -86,6 +87,9 @@ export const LogLineContext = memo(
     const [aboveState, setAboveState] = useState(LoadingState.NotStarted);
     const [belowState, setBelowState] = useState(LoadingState.NotStarted);
     const [showLog, setShowLog] = useState(false);
+    const [datasourceInstance, setDatasourceInstance] = useState<
+      (DataSourceApi & { hasLogsContextAdjustableWindow?: boolean }) | null
```

Comment:
Probably:

```suggestion
      (DataSourceApi & DataSourceWithLogsContextSupport) | null
```

Reply:
Fixed in https://github.com/grafana/grafana/pull/109934/commits/9ee0cc812869f5c62cf5680111dfd71f24144bb6